ubtue_Brill.js

```diff
@@ -109,7 +109,9 @@ function postProcess(doc, item) {
 	// mark articles as "LF" (MARC=856 |z|kostenfrei), that are published as open access	
 	let openAccessTag = text(doc, '.has-license span');
 	if (openAccessTag && openAccessTag.match(/open\s+access/gi)) item.notes.push('LF:');
-  
+  // mark articles as "LF" (MARC=856 |z|kostenfrei), that are free accessible e.g. conference report 10.30965/25890433-04902001 
+	let freeAccess = text(doc, '.color-access-free');Z.debug(freeAccess)
+	if (freeAccess && freeAccess.match(/(free|freier)\s+(access|zugang)/gi)) item.notes.push('LF:');
```

Comment:
Cf. line 111: Make sure `LF:` is inserted only once by using a `if .. else` construct?

Reply:
yes, it is either open access or free access, therefore "LF:" is only assigned once

Reply:
Yes, but more cases may occur in the future. Will merge it anyway.